### PR TITLE
Add clipper.exchange on Optimism

### PR DESCRIPTION
**Pull request by amalashkevich**
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [x] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [x] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [x] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [x] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you

[Sample clipper_v1_optimism_trades](https://dune.com/queries/2459708)
[Sample clipper_v2_optimism_trades](https://dune.com/queries/2459707)
[Sample clipper_coves_v1_optimism_trades](https://dune.com/queries/2459698?d=1)

**Comment by amalashkevich** [comment]
Thank you, @Hosuke

**Comment by Hosuke** [comment]
Hi @amalashkevich there may be some conflicts between branches need to be resolved.
image

Please sync/update the branch with main manually.

**Review by Hosuke — APPROVED** [review]

File: models/clipper/optimism/clipper_optimism_schema.yml
Comment (Hosuke) [inline models/clipper/optimism/clipper_optimism_schema.yml:171]: Here is a typo:
```suggestion
      - check_dex_seed:
          blockchain: optimism
          project: clipper
          version: coves1
```

Comment (amalashkevich) [reply models/clipper/optimism/clipper_optimism_schema.yml]: Thanks, fixed

**Review by Hosuke — APPROVED** [review]
LGTM. for the changes

**Comment by amalashkevich** [comment]
Hi @Hosuke 
Could you please share when it could be merged? The deprecated queries do not show fresh data, so we need this spell.

**Comment by Hosuke** [comment]
> Hi @Hosuke Could you please share when it could be merged? The deprecated queries do not show fresh data, so we need this spell.

Hi @amalashkevich can you update the branch? We need to wait for a second reviewer for the final review.
Probably this PR will be merged in 2 days.

**Review by jeff-dude — APPROVED** [review]
thanks all

**Pull request merged**
